README.md

```diff
@@ -24,17 +24,24 @@ This module provides StackDriver Trace support for Node.js applications. [StackD
 
         npm install --save @google/cloud-trace
 
-2. Include and start the library at the *top of the main script of your application*. It's important that the trace agent is the first thing executed so that it can accurately gather data:
+2. Set the GCLOUD_PROJECT environment variable. You can find your Project ID in the [Google Cloud Developers Console](https://console.cloud.google.com/project?_ga=1.258049870.576536942.1443543237), or by running the command `gcloud projects list`. You can ensure this environment variable is set at startup time by placing it in your startup script in `package.json`:
```

Comment:
Can you use the out-of-line markdown link syntax and put the link at th bottom; and drop the `?_ga=...`